code/game/objects/effects/spiders.dm

```diff
@@ -210,25 +210,8 @@
 	//=================
 
 	if(isturf(loc))

```

Comment:
Can't you cut off the this bit here too, since skitter checks this?

Reply:
No, since there's an else-if that requires this if proc afterward.